lib/base/base.js

```diff
@@ -1849,7 +1855,7 @@ $.widget("sv.widget", {
 */
 
 	/**
-	* TODO: check if partial update is still neded here. With smarthome.py the complete series is transferred
+	* TODO: check if partial update is still needed here. With smarthome.py the complete series is transferred
```

Comment:
smarthome.py or SmartHomeNG? BTW: Is there anywhere still support for the old smarthome.py?

Reply:
Both is correct. The drivers filename is still „io_smarthome.py.js” which I don't want  to change.  BTW: the filename of shNG itself is „smarthome.py“, too ;)  The driver will  not work with older versions which require the websocket protocol v3 since this has been deprecated and removed.